ansible/vars/icds/icds_public.yml

```diff
@@ -36,7 +36,7 @@ active_sites:
   riakcs: True
   cas_ssl: True
 
-riak_ring_size: 128
+riak_ring_size: 64
```

Comment:
woah! I thought ICDS had ring size of 128. I don't think this should change, and I'm not understanding why you had to change it now.